### ISSUE TITLE
Laravel 5.3

### ISSUE THREAD

**Issue by stojankukrika** [maddhatter/laravel-fullcalendar#53]
Can you add support for laravel 5.3?

**Comment by maddhatter** [comment]
I haven't had much time to work on my Laravel projects in general; unfortunately, I don't foresee that getting added any time in the near future (by me at least).

I'd be happy to merge in a PR that adds 5.3 support - I'm not even sure what would need to be changed.

**Comment by joaoguilhermels** [comment]
Hi ! I am using laravel-fullcalendar on a 5.3 project and it is working. I'll check what can be improved because of 5.3 changes and let you know.

**Comment by ThamrinPH** [comment]
@joaoguilhermels can you share your project pls, i can't even make the calendar shows up

**Comment by joaoguilhermels** [comment]
@ThamrinPH sorry, I can't share my project. Are you following the documentation? Are you seeing any errors?

**Comment by ThamrinPH** [comment]
yeah, i followed the documentation on basic usage section, i got the script
right and the css too, i checked it leads to the right file, and the
calendar didn't show up at part "And that's it, you should see a
month-based calendar that has no events on it. " . . . is there something
that i missed, like installing something or require something first?

Pada tanggal 16 Des 2016 05:13, "João Guilherme" <notifications@github.com>
menulis:

**Comment by darksyfer** [comment]
@ThamrinPH hi. what did you do to get this right? All the files get loaded correctly but could not figure it out why the calendar does not show up. Thank you.
